plugins/kontent-ai-plugin/package.json

```diff
@@ -8,15 +8,15 @@
     "setup": "node ./scripts/setup.js"
   },
   "dependencies": {
-    "@kontent-ai/gatsby-source": "9.0.1",
-    "@kontent-ai/gatsby-components": "9.0.1",
-    "gatsby-plugin-manifest": "^4.7.0",
-    "uuid": "^8.3.2"
+    "@kontent-ai/gatsby-source": "10.0.0",
+    "@kontent-ai/gatsby-components": "10.0.0",
+    "gatsby-plugin-manifest": "^5.4.0",
```

Comment:
Okay one more tweak — I'm seeing an error relating to the version of `sharp` with `5.4.0` of `gatsby-plugin-manifest`. That same error is not present when we use `5.0.0`, which is what the rest of the flavors are currently on as well. I'll notify the framework team of this issue.